commitlint.config.js

```diff
@@ -4,10 +4,7 @@
 // License text available at https://opensource.org/licenses/MIT
 
 module.exports = {
-  extends: [
-    '@commitlint/config-conventional',
-    '@commitlint/config-lerna-scopes',
-  ],
+  extends: ['@commitlint/config-conventional', './bin/config-lerna-scopes'],
```

Comment:
I know the switch to away from `@commitlint/config-lerna-scopes` is because it doesn't yet support Lerna 3.x **BUT** they have a PR in progress to add 3.x support ... I don't think we should introduce our own script for the package as it becomes something for us to maintain. We can split this PR to land the TypeScript upgrade and give `commitlint` some time to upgrade (knowing their history and the current progress of the PR I'd say a week tops) and we won't have to have a custom script to maintain?

Reply:
commitlint PR: https://github.com/marionebl/commitlint/pull/406

Reply:
Haha, I didn't see the PR and was trying to submit one. It's actually not too bad to maintain some of the commit rules for `loopback-next` itself though, IMO.